charts/midaz/templates/crm/configmap.yaml

```
@@ -10,6 +10,7 @@ data:
   ENV_NAME: {{ .Values.crm.configmap.ENV_NAME | default "development" | quote }}
 
   # APP
+  APPLICATION_NAME: {{ .Values.crm.configmap.APPLICATION_NAME | default "ledger" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)